server/service/handler.go

```diff
@@ -787,6 +787,13 @@ func attachFleetAPIRoutes(r *mux.Router, svc fleet.Service, config config.FleetC
 	// input to `fleetctl apply`
 	ue.POST("/api/_version_/fleet/mdm/profiles/batch", batchSetMDMProfilesEndpoint, batchSetMDMProfilesRequest{})
 
+	// Certificate management
+	ue.GET("/api/_version_/fleet/certificate_mgmt/certificates", getCertificatesEndpoint, nil)
```

Comment:
```suggestion
	ue.GET("/api/_version_/fleet/pki", getCertificatesEndpoint, nil)
```

`/pki` to be consistent w/ the UI?

![Screenshot 2024-11-21 at 4 07 52 PM](https://github.com/user-attachments/assets/b0afeec5-4d2e-4638-81e6-02abde20f712)

Reply:
ok